### ISSUE TITLE
Availability of model function in template

### ISSUE THREAD

**Issue by renestalder** [amteich/kirby-twig#21]
So, I browsed through the documentation and quickly tried whether a function define in a Kirby model is available in the Twig templates, but It doesn't seem to be.

Is there any way to use regular Kirby page models with the Twig plugin?

**Comment by seehat** [comment]
I think that it should work to call page model methods. For examle a custom model function of a page. 

Could you provide an example, which is not working for you?

**Comment by renestalder** [comment]
Turns out, my mistake was that I tried it with the default template, which of course doesn't map to the content file e.g. `models/default.php` will not work. After your answer, I started replicating the test with the homepage template, the php version and the Twig version, I now can tell that it was a mistake in how I applied it before.

So, this works flawlessly.

`site/models/home.php`:

```php
<?php

// For a content file called `project.txt`
// In general the class name is {{ProjectFileName}}Page

class HomePage extends Page {
  public function test() {
    return "Test";
  }
}
```

`site/templates/home.twig`:

```twig
{{ page.test }}
```

Sorry for stealing your time, and thanks for confirming, that it works.
